src/convert.spec.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Looker Data Sciences, Inc.
```

Comment:
Does Intellij have a new file template feature somewhere?  It's getting tedious to copy this header around in new files.  Plus, update the year for new files

Reply:
Oh, good point on year. Nope, I'm just copy/pastaing .. and have a wishlist item to inject the license in all source files where it's missing w/ a command-line script. We need something that will work for all source file types, of course.